ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master.yaml

```diff
@@ -69,6 +69,10 @@ tests:
   commands: make manifest/check
   container:
     from: src
+- as: version
```

Comment:
Might be better called versions?

Reply:
Sure, I thought it was versions already, must have mistyped